### PR TITLE
Change cm DNS records from sn05 to sn06

### PR DESCRIPTION
**Pull request by sanjaysrikakulam**
As part of DB server migration HTCondor central manager will be deployed on sn06.

Relevant PR: https://github.com/usegalaxy-eu/infrastructure-playbook/pull/783
DB server migration issue: https://github.com/usegalaxy-eu/issues/issues/436

### CODE REVIEW

File: instance_core_maintenance.tf [hunk 1]
```diff
@@ -7,7 +7,7 @@ resource "openstack_compute_instance_v2" "maintenance" {
   flavor_name     = "m1.xlarge"
   key_pair        = "cloud2"
   tags            = []
-  security_groups = ["default"]
+  security_groups = ["default", "rsyslog"]
```
Comment (kysrpex) [inline instance_core_maintenance.tf:10]: Is this change intended? Or did it slip from some file you were working on for the maintenance node?

Comment (sanjaysrikakulam) [reply instance_core_maintenance.tf:10]: Yes, can be a separate PR though. Since it's not a big change I added it here to avoid Terrafrom removing the manually attached `rsyslog` security group.

**Review by kysrpex — APPROVED** [review]
